integration/update_field_compat_test.go

```diff
@@ -52,34 +52,34 @@ func TestUpdateFieldCompatCurrentDate(t *testing.T) {
 		},
 		"BoolTrue": {
 			update: bson.D{{"$currentDate", bson.D{{"v", true}}}},
-			paths:  []types.Path{types.NewPathFromString("v")},
+			paths:  []types.Path{must.NotFail(types.NewPathFromString("v"))},
```

Comment:
I guess `NewStaticPath` should also work for all those changes with `must.NotFail`, but I don't think it's critical.

Reply:
Makes sense, changed it.